client/src/components/ComboBox/ComboBox.scss

```diff
@@ -28,9 +28,9 @@ $width: min(1000px, 75vw);
 
   // Case 1: base case
   // Use max-content width to minimize shifting due to content wrapping,
-  // while also ensuring it's not wider than 75% of the viewport.
+  // constraining width to 75% of viewport for devices above 400px width.
   width: max-content;
-  max-width: 75vw;
+  max-width: max(300px, 75vw);
```

Comment:
Unrelated to other changes – fixes a small layout issue with contents of the combobox overflowing their container on screen widths in the 320-375px range.